### PR TITLE
Add Runkeeper

### LINKED ISSUE #3929: Please add Runkeeper
Please add [Runkeeper](https://runkeeper.com). They don't support 2FA.

### PR DESCRIPTION
**Pull request by rakleed**
Fix https://github.com/2factorauth/twofactorauth/issues/3929

### CODE REVIEW

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Here's a better logo:

**Comment by rakleed** [comment]
@Carlgo11 done. And should I write such messages as this, or do you get a notification after I add the right file?

**Comment by Carlgo11** [comment]
@Rakleed I receive an email when you change something however it’s alright if you want to write a message about it as well. Many people do that so it’s not really a problem. 🙂
A02EE901-61E3-4F42-856E-8AA44B2BE3C7

File: _data/health.yml [hunk 1]
```diff
@@ -117,6 +117,14 @@ websites:
       img: myfitnesspal.png
       tfa: No
 
+    - name: Runkeeper
+      url: https://runkeeper.com
+      twitter: Runkeeper
```
Comment (Carlgo11) [inline _data/health.yml]:
```suggestion
      twitter: RKSupport
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Review by kmpoppe — APPROVED** [review]

**Pull request merged**